Stitch/App/StitchApp.swift

```diff
@@ -38,7 +38,13 @@ struct StitchApp: App {
                         options.debug = false
                     }
                     
-                    FirebaseApp.configure()
+                    #if !DEBUG
```

Comment:
This needs to be #if RELEASE
```suggestion
                    #if RELEASE
```

Reply:
This also isn't enough because this still won't work if folks want to run release builds locally. This is a valid use case.